pkg/auth/server/grant/templates.go

```diff
@@ -9,8 +9,9 @@ const defaultGrantTemplateString = `<!DOCTYPE html>
 <html>
 
 <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>
-      Authorize 
+      Authorize
```

Comment:
probably want to be careful about these whitespace changes, it might be affecting the final output